src/lib/Repository/UiLocationService.php

```diff
@@ -52,14 +63,38 @@ public function loadLocations(ContentInfo $contentInfo)
         return $uiLocations;
     }
 
+    /**
+     * Deletes locations.
+     * Throws exception if there is an attempt to delete the main location.
+     *
+     * @param array $locationIds
+     * @param int $contentId
+     *
+     * @throws ForbiddenException
+     */
+    public function deleteLocations(array $locationIds, int $contentId)
```

Comment:
We may want to rename it to `deleteSecondaryLocations()`, as it won't delete a main location.